Formula/libvirt.rb

```diff
@@ -14,6 +14,7 @@ class Libvirt < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnutls"
+  depends_on "curl"
```

Comment:
Any version seems to work.  It's just that the default curl doesn't seem to be found for some unknown reason (I didn't investigate), whereas a Homebrew provided one does.